linkerd/buffer/src/dispatch.rs

```diff
@@ -29,7 +29,7 @@ pub(crate) async fn run<S, Req, I>(
             req = requests.recv().fuse() => {
                 match req {
                     None => return,
-                    Some(InFlight { request, tx }) => {
+                    Some(InFlight { request, tx, .. }) => {
```

Comment:
nit: Are there other fields in `InFlight` now?
```suggestion
                    Some(InFlight { request, tx }) => {
```

Reply:
no, there was a permit here before the channel was factored out to also be used in `discovery::buffer`, but i forgot to change this back --- good eye!